tests/unit/openwrt/golden/ipq806x/root/bin/config-version.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+usage(){
+  cat << EOF
+usage: "$(basename "$0")" [OPTIONS] ARGS
```

Comment:
if there is no config version specified, it should return the config in use (and possibly the filename list??)

Reply:
:+1: I can see that as better behavior, I think given that the filename list is static it should just return the config version that's in use

Reply:
I think @sarcasticadmin has the right idea. Let's do that.